server/testing-docker-compose.yml

```diff
@@ -10,13 +10,13 @@ services:
       - "5432:5432"
 
   redis:
-    image: "redis:7-alpine"
+    image: "redis:7.0.10-alpine"
```

Comment:
I don't think you need to touch this one, since it appears to be a bitnami-specific change, but all good.

Reply:
Yeah, I hear you, but I feel better seeing the same version in lock-step down the file.